tests/test_unexpansion.py

```diff
@@ -22,7 +22,7 @@ def test_backward_dt2(self):
         assert_structure(op, ['t,x,y'], 't,x,y')
 
 
-class TestSymbolicCoefficients(object):
+class TestSymbolicCoeffs(object):
```

Comment:
Think we should also test the case where different coefficients are passed for x and y derivatives with cross-derivatives, mixed derivatives, and non-perfect mixed derivatives. Also the case where only one of the x or y derivatives gets coefficients specified but bother are defined as symbolic.